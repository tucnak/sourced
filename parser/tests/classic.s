bind ALT "say \"Classic Source bind style\";reload"
