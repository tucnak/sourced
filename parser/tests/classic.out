alias bind_ALT "say "Classic Source bind style";reload";bind ALT bind_ALT
